KomodoPlatform/atomicDEX-Desktop
# fix infinite fees loading issue

closes https://github.com/KomodoPlatform/atomicDEX-Desktop/issues/2112
- Removes extra `trade_preimage` rpc calls
- Removes extra `cap_volume` rpc calls (slightly faster simple mode loading)

To test: 
- select different orders in quick succession. Fees should display, not attempt to reload (no busy indicator after fees shown) and not get in a state of perpetual busy indicator on both pro and simple mode
